.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,13 @@
+name: Lint GitHub Actions Workflows
+on:
+  push:
+  pull_request:
+permissions:
+  contents: read
+jobs:
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
```

Comment:
dependabot is now notifying to update actions/checkout from 3.3.0 to 3.5.0, might be a good idea to pin all references to actions/checkout to 3.5.0 before merging.